tests/e2e/suite_test.go

```diff
@@ -469,10 +470,12 @@ var _ = BeforeSuite(func() {
 		sequential = false
 	}
 
-	// Same for isoBoot
-	switch isoBootString {
-	case "true":
+	// Same for bootType
+	switch bootTypeString {
+	case "iso":
 		isoBoot = true
+	case "raw":
+		rawBoot = true
 	default:
```

Comment:
In fact boolean type is set to false by default, so the `default` for `isoBoot` can be removed.